Makefile

```diff
@@ -7,6 +7,7 @@ BASE_IMAGE := ubuntu:24.04
 DOCKER_IMAGE := docker/model-runner:latest
 PORT := 8080
 MODELS_PATH := $(shell pwd)/models-store
+GPU_DEVICE_FLAG := $(shell if [ -d /dev/dri ]; then echo "--device /dev/dri"; fi)
```

Comment:
Tempting to make part of the makefile call a shell script to avoid these funny syntaxes